invenio_github/errors.py

```diff
@@ -32,18 +32,61 @@ class GitHubError(Exception):
 class RepositoryAccessError(GitHubError):
     """Repository access permissions error."""
 
+    def __init__(self, user=None, repo=None, repo_id=None):
+        """Constructor."""
+        self.user = user
+        self.repo = repo
+        self.repo_id = repo_id
+
+    def __str__(self):
+        """String."""
+        return u'The user cannot access the github repository'
```

Comment:
I think a better option is to pass a default message to `super(...).__init__(...)` and rely on keyword arguments only (as done already in this PR) for capturing specific info:

```python
class RepositoryAccessError(GitHubError):
    """Repository access permissions error."""

    message = u'The user cannot access the github repository'

    def __init__(self, user=None, repo=None, repo_id=None, message=None):
        """Constructor."""
        super(RepositoryAccessError, self).__init__(message or self.message)
        self.user = user
        self.repo = repo
        self.repo_id = repo_id

    # Note: No need for __str__ overriding
```